lib/eosknowledgeprivate/ekn-resources.gresource.xml

```diff
@@ -2,5 +2,7 @@
 <gresources>
   <gresource prefix="/com/endlessm/knowledge-private">
     <file preprocess="xml-stripblanks">EknRuntimeDocumentViewer.ui</file>
+    <file preprocess="xml-stripblanks">EknMediaBin.ui</file>
```

Comment:
I am linking statically with this resource, so if I set it to compressed then we would end up with the compressed data in the text area and the decompresed data in the heap.

So we either waste some disk space or some memory :S